src/Symfony/Bundle/FrameworkBundle/Test/WebTestAssertions.php

```diff
@@ -0,0 +1,383 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use PHPUnit\Framework\Assert;
+use Symfony\Bundle\FrameworkBundle\Client;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Ideas borrowed from Laravel Dusk's assertions.
+ *
+ * @see https://laravel.com/docs/5.7/dusk#available-assertions
+ */
+trait WebTestAssertions
+{
+    /** @var Client|null */
+    protected static $client;
+
+    public static function assertResponseIsSuccessful(): void
+    {
+        $response = static::getResponse();
+
+        Assert::assertTrue(
+            $response->isSuccessful(),
+            \sprintf('Response was expected to be successful, but actual HTTP code is %d.', $response->getStatusCode())
+        );
+    }
+
+    public static function assertHttpCodeEquals(int $expectedCode): void
+    {
+        Assert::assertSame(
+            $expectedCode,
+            $code = static::getResponse()->getStatusCode(),
+            \sprintf('Response code "%s" does not match actual HTTP code "%s".', $expectedCode, $code)
```

Comment:
We don't need the `\\` before `sprintf`, we are only using it when it makes a performance difference, which is not the case here.

Reply:
Done.

Is there any doc with the list of functions that are concerned by a performance boost with this backslash?

Reply:
Here is the issue that references them: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048
The process is the following: never add any, then let php-cs-fixer/fabbot add them where needed.